src/modules/Header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import './scss/Header.scss';
+
+function Header() {
+  return (
+    <div>
+        <nav>
+            <Link className="title_link" to="/">Bookstore CMS</Link>
+            <Link className="link" to="/books">BOOKS</Link>
+            <Link className="link" to="/categories">CATEGORIES</Link>
```

Comment:
✅ Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 👍🏻